src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -107,32 +109,25 @@ def get_hostname_completion_list(prefix, action, parsed_args, **kwargs): # pylin
-register_cli_argument('appservice web config backup restore', 'storage_blob_name', help='Name of the backup blob to restore (e.g. backup.zip)')
-register_cli_argument('appservice web config backup restore', 'database_name', help='Name of the database to restore')
-register_cli_argument('appservice web config backup restore', 'database_type', help='Type of database to restore', **enum_choice_list(database_types))
-register_cli_argument('appservice web config backup restore', 'database_connection_string', help='Connection string of database server for the restored database')
-register_cli_argument('appservice web config backup restore', 'target_name', help='Name of the app which will be overwritten with the restored data')
-register_cli_argument('appservice web config backup restore', 'overwrite', help='Must be true to allow the source web app to be overwritten')
-register_cli_argument('appservice web config backup restore', 'ignore_hostname_conflict', help='If true, ignores custom hostnames stored in the backup')
+db_types = [DatabaseServerType.my_sql, DatabaseServerType.sql_azure]
+frequency_units = [FrequencyUnit.hour, FrequencyUnit.day]
+
+register_cli_argument('appservice web config backup', 'storage_account_url', help='URL with SAS token to the blob storage container', options_list=['--container-url'])
+register_cli_argument('appservice web config backup', 'webapp_name', help='The name of the webapp')
+register_cli_argument('appservice web config backup', 'db_name', help='Name of the database in the backup', arg_group='Database')
+register_cli_argument('appservice web config backup', 'db_connection_string', help='Connection string for the database in the backup', arg_group='Database')
+register_cli_argument('appservice web config backup', 'db_type', help='Type of database in the backup', arg_group='Database', **enum_choice_list(db_types))
+
+register_cli_argument('appservice web config backup create', 'backup_name', help='Name of the backup. If unspecified, the backup will be named with the webapp name and a timestamp')
+
+register_cli_argument('appservice web config backup update', 'frequency', help='How often to backup. Use a number followed by d or h, e.g. 5d = 5 days, 2h = 2 hours')
+register_cli_argument('appservice web config backup update', 'keep_at_least_one_backup', help='Always keep one backup, regardless of how old it is', options_list=['--retain-one'], action='store_true')
+register_cli_argument('appservice web config backup update', 'retention_period_in_days', help='How many days to keep a backup before automatically deleting it. Set to 0 for indefinite retention', options_list=['--retention'])
+
+register_cli_argument('appservice web config backup restore', 'backup_name', help='Name of the backup to restore')
+register_cli_argument('appservice web config backup restore', 'target_name', help='Name of the app which will be overwritten with the restored data. If unspecified, the target is the source webapp')
```

Comment:
Possibly "The name to use for the restored webapp. If unspecified, will default to the name that was used when the backup was created."